pkg/diff/util.go

```diff
@@ -24,6 +28,20 @@ import (
 	"go.uber.org/zap"
 )
 
+var (
+	successState = "success"
+
+	failedState = "failed"
+
+	errorState = "error"
```

Comment:
add some comments for these `state`? what's differences between `failedState` and `errorState`?

Reply:
add comment